store/copr/coprocessor_cache.go

```diff
@@ -60,6 +61,10 @@ func newCoprCache(config *config.CoprocessorCache) (*coprCache, error) {
 	if config == nil || config.CapacityMB == 0 {
 		return nil, nil
 	}
+	if config.CapacityMB < 0 {
+		logutil.BgLogger().Info("Capacity should >= 0")
+		return nil, errors.New("Capacity should >= 0")
+	}
```

Comment:
The error message is not clear enough for users. How about just merge this part into the judgement below?
`if capacityInBytes == 0` -> `if capacityInBytes <= 0`.